README.md

```diff
@@ -1,46 +1,88 @@
+[Live Demo Link]( https://anasbassi.github.io/Crypto-Stat/)
 
-### `npm test`
+## Description video ✔️
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[Loom Video Link]()
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Getting Started 🙌
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+To get a local copy up and running follow these simple example steps.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Prerequisites
+```
+  node.js
+  npm
 
-### `npm run eject`
+```
+### Setup
+Clone the repository and change into the  directory Crypto-Stat
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+``` 
+  $ git clone 'https://github.com/anasbassi/Crypto-Stat.git'
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+  $ cd FX-Today
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+```
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+### Install
+Install all project dependencies by running the command below
+ 
+``` 
+ $ npm install
+```
+### Usage
 
-## Learn More
+Run
+```
+  $ npm start
+```
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### Deployment
+- http://localhost:3000
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+
+## Authors
+
+👤 **Anas Abubakar Bassi**
+
+- GitHub: [@anasbassi](https://github.com/anasbassi)
+- Twitter: [@anasabu92568449](https://twitter.com/anasabu92568449)
+- LinkedIn: [Anas Abubakar](https://linkedin.com/in/anas-abubakar-7b352722b)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/anasbassi/Crypto-Stat/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Microverse Community
+- [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) (For the App design)
```

Comment:
- [ ] Good job to give then of the design author :+1:  But please add the link of [ the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) :smile: